src/data/pokemon/species_info/gen_7_families.h

```diff
@@ -1197,7 +1197,7 @@ const struct SpeciesInfo gSpeciesInfoGen7[] =
         .catchRate = 45,
         .expYield = 167,
         .evYield_SpAttack = 2,
-        .itemRare = ITEM_HONEY,
```

Comment:
We could consider adding configs for this rather than just outright replacing the previous held items

Reply:
We should probably make a bigger PR for configs for all gens instead of just doing it here. So we can do it later.